tests/legacy/system/test_edb.py

```diff
@@ -1732,3 +1730,91 @@ def test_multizone(self, edb_examples):
         assert defined_ports
         assert project_connexions
         edbapp.close_edb()
+
+    @pytest.mark.skipif(not desktop_version == "2024.2", reason="Only supported with 2024.2 and higher")
```

Comment:
```suggestion
    @pytest.mark.skipif(not desktop_version == "2024.2", reason="Only supported with 2024.2 and higher")
```
Here you won't cover 2025.1, please update the skipif.

Reply:
Done